src/main/java/dev/ebyrdeu/movie_jakarta/entity/Movie.java

```diff
@@ -0,0 +1,64 @@
+package dev.ebyrdeu.movie_jakarta.entity;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+
+import java.util.Objects;
+import java.util.UUID;
+
+@Entity
+public class Movie {
+    @Id
+    @GeneratedValue(strategy = GenerationType.UUID)
+    private UUID id;
+    private String title;
+    private int releaseYear;
+    private String director;
```

Comment:
I have a few concerns about this. However, since we have just one simple entity, the Hibernate mapper should work quite well here. Good job.